app/Http/Controllers/ForwardChatworkController.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Webhook;
+use App\Models\Condition;
+use Illuminate\Http\Request;
+
+class ForwardChatworkController extends Controller
+{
+    public function forwardMessage(Request $request, $token)
+    {
+        // convert array to object with multiple level
+        $params = json_decode(json_encode((object) $request->all()), FALSE);
+        $webhook = Webhook::where('token', '=', $token)->first();
+        $payloads = $webhook->payloads;
+        $filled_payloads = [];
+
+        foreach ($payloads as $payload) {
+            $conditions = $payload->conditions;
+            $isValid = true;
+
+            foreach ($conditions as $condition) {
+                $expression = 'return ' . $condition->field . ' '. $condition->operator . ' "' . $condition->value . '";';
+                if (!eval($expression)) {
+                    $isValid = false;
+                    break;
+                }
+            }
+
+            if ($isValid) {
+                array_push($filled_payloads, $payload);
+            }
+        }
+
+        // dd(collect($filled_payloads)->pluck('content'));
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Http/Controllers/ForwardChatworkController.php</br>**Line**: 38</br>Variable "filled_payloads" is not in valid camel caps format